scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -1127,11 +1118,15 @@ mod tests {
                 .unwrap(),
         );
         setup_tables(&session, &schema).await.unwrap();
-        execute_queries(&session, operations).await.unwrap();
 
-        replicate(&session, &schema.name).await.unwrap();
+        session.use_keyspace("test_src", false).await.unwrap();
+
+        for operation in operations {
+            session.query(operation.0, operation.1).await.unwrap();
+            replicate(&session, &schema.name).await.unwrap();
```

Comment:
Here, too.

Reply:
As you see, we call `replicate` in at least two places. If we store somehow the last read `cdc$time` won't we get some race conditions after we parallelize the tests?